ocaml/parsing/pprintast.ml

```diff
@@ -1468,16 +1468,16 @@ and signature_item ctxt f x : unit =
       pp f "@[<hov2>module@ type@ %s%a@]%a"
         s.txt
         (fun f md -> match md with
-           | None -> ()
-           | Some mt ->
+           | Pmtd_abstract | Pmtd_underscore -> ()
```

Comment:
`pprintast` is a pretty printer that should reproduce user input (but prettier). So here `Pmtd_underscore` should produce underscore.